iceoryx2/src/service/static_config/mod.rs

```diff
@@ -37,28 +37,39 @@ pub struct StaticConfig {
     pub(crate) messaging_pattern: MessagingPattern,
 }
 
+fn create_uuid<Hasher: Hash>(
+    service_name: &ServiceName,
+    messaging_pattern: &MessagingPattern,
+) -> Hasher {
+    let pattern_and_service = (<MessagingPattern as Into<u32>>::into(messaging_pattern.clone()))
```

Comment:
Is there a reason to not have the `MessagingPattern` implement `Copy`?

Just a general question, no need to change it in this PR if you agree to make it `Copy`.

Reply:
I would add this in the last of the 3 PRs

Reply:
@elBoberido no it is not possible since the static config contains a string which is not copyable.

Reply:
My bad. I assumed it was a simple number enum